autogpt_platform/frontend/src/app/(platform)/library/agents/[id]/page.tsx

```diff
@@ -246,6 +275,10 @@ export default function AgentRunsPage(): React.ReactElement {
               href: `/build?flowID=${agent.graph_id}&flowVersion=${agent.graph_version}`,
             },
             { label: "Export agent to file", callback: downloadGraph },
+            {
+              label: "Edit a copy",
```

Comment:
```suggestion
              label: "Create & edit a copy",
```
or

```suggestion
              label: "Copy & Edit",
```

cc @ograce1421

Reply:
imo none of these, including original, are accurate - we're copying (cloning) and viewing, not editing. How about `Make a copy` or `Copy agent` or `Make editable copy` (because marketplace ones are editable)?

Reply:
We're keeping it as is - according to the design

Reply:
> we're copying (cloning) and viewing, not editing

> marketplace [agents] are not editable

vs

> Let's keep it as how it is in the Figma File: "Edit a copy"

> This is to help users to quickly understand what action they can do / what they get if they press on the button

This doesn't seem to add up to me. @ograce1421 I assume that the button caption being accurate trumps the need for simplicity, right? We either need to change the behavior or the caption. Otherwise it gives the false impression that they can edit non-editable agents. 

Also, @kcze am I understanding correctly that marketplace agents *can* be forked, but the resulting forks *aren't* editable? If that's the case, what's the point of letting users fork them?

Reply:
In that case "Edit a copy" is misleading imo